### PR TITLE
Using google maps #18

### PR DESCRIPTION
**Pull request by ivinjose**
@shidhincr  This replaces the background with actual map.

### CODE REVIEW

File: app/scripts/ui/Map.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    },
+    getMapCenter: function(){
+        return new google.maps.LatLng(this.props.centerLat,this.props.centerLng);
+    },
+    componentDidMount: function(rootNode){
+        var mapSettings = {
+            center: this.getMapCenter(),
+            zoom: this.props.initialZoom
+        };
+        //TODO:: Ivin: Couldnt find a way to pass the dom reference from the place of usage. Hardcoding for now.
```
Comment (ivinjose) [inline app/scripts/ui/Map.js]: @shidhincr  Can you think of a way to do this?

**Comment by shidhincr** [comment]
@ivinjose This looks so nice.

File: app/scripts/ui/Map.js (hunk 1 above)
Comment (shidhincr) [inline app/scripts/ui/Map.js]: @ivinjose check this http://facebook.github.io/react/docs/working-with-the-browser.html

I made the changes and pushed to this branch. Please refer the code changes

**Comment by shidhincr** [comment]
@ivinjose I am merging this brach.. Please check the changes I have done also.

**Pull request merged**